mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
I don't understand where these magic values are coming from.  Should they be documented? Do they have any significance?  If they're not significant, could they just be made random instead of static?